index.html

```diff
@@ -947,620 +974,1301 @@
         </div>
       </div>
 
-      <div class="container d-lg-none">
+      <div class="container d-none d-lg-block main2 mt-5 pt-5">
         <div class="row">
-          <div class="col-12 hidden mit">
-            <div class="featured">TOP STORY</div>
-            <div class="relative-img">
-              <img src="./assets/mitt-romney.jpg" alt="mit" />
-              <div class="square rom">U.S</div>
+          <div class="col-lg-12 in-magazine">
+            <div class="row in-magazine-title">
+              <div
+                class="col-lg-12 d-lg-flex border-bottom pb-1 border-secondary justify-content-center"
+              >
+                <img
+                  class="img-fluid n-logo-small"
+                  src="./assets/n-logo-small.JPG"
+                  alt="Newsweek logo small"
+                />
+                <strong class="ml-2">IN THE MAGAZINE</strong>
+              </div>
             </div>
+            <div class="row in-magazine-row1">

```

Comment:
- [ ] This section should have it's title in a header tag, h2, h3, h4, etc, instead of a strong tag, and use the semantic section tag instead of a div.
Please go through each new section you added in this milestone and at the previous and make sure all of them have the semantic section tag and the title in a header, h2, h3, h4, etc, tag.